picard/album.py

```diff
@@ -54,7 +57,6 @@ def __init__(self, album_artist_id):
 
 
 class Album(DataObject, Item):
-
```

Comment:
Now i think it is wrong, because of https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

> Insert a blank line after all docstrings (one-line or multi-line) that document a class -- generally speaking, the class's methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line.


```python
class TheClass(object):

   def method(self):
      ...
```

```python
class TheClass(object):
   """docstring"""

   def method(self):
      ...
```

```python
class TheClass(object):
   """docstring"""

   somevar = 123

   def method(self):
      ...
```

```python
class TheClass(object):

   somevar = 123

   def method(self):
      ...
```